pkg/minikube/audit/row_test.go

```diff
@@ -126,7 +126,7 @@ func TestRow(t *testing.T) {
 			{"user", r.user, u},
 			{"version", r.version, v},
 			{"startTime", r.startTime, stFormatted},
-			{"endTime", r.endTime, etFormatted},
+			{"id", r.id, id},
```

Comment:
@spowelljr 
Made changes as suggested
What can be the reason for the test fails ? 
Could you please review the code
- I think the changes I made in this (`row_test.go`) file can be the reason for it. Here I replaced `endTime` with `id` because parameter `endTime` was changed in `newRow()` func by `id`.

Reply:
`TestAudit/LogCommandStart` is failing because you're only passing one arg
```
os.Args = []string{"minikube"}
```
But the func has:
```
if len(os.Args) < 2 || !shouldLog() {
    return "", nil
}
```

Also, you haven't updated `assignFields()` and `toMap()` to use the new `id` field
```
No "id" field below

func (e *row) toMap() map[string]string {
	return map[string]string{
		"args":      e.args,
		"command":   e.command,
		"endTime":   e.endTime,
		"profile":   e.profile,
		"startTime": e.startTime,
		"user":      e.user,
		"version":   e.version,
	}
}
```

Reply:
@spowelljr 
Made changes as suggested
```
- Now `TestAudit/LogCommandStart` is failing and saying that `audit ID should not be empty`. 
- So should I make a condition in `LogCommandStart()` for empty id. I am not very sure about this because in few conditions in the function like

if len(os.Args) == 0 || !shouldLog() {
		return "", nil
}
 we are returning empty string, so how can we deal with it ? 
```

Reply:
@spowelljr 
Made changes as suggested
```
- Now `TestAudit/LogCommandStart` is failing and saying that `audit ID should not be empty`. 
- So should I make a condition in `LogCommandStart()` for empty id. I am not very sure about this because in few conditions in the function like

if len(os.Args) == 0 || !shouldLog() {
		return "", nil
}
 we are returning empty string, so how can we deal with it ? 
```

Reply:
You need to increase the number of args to 2.
Right now you have:
```
os.Args = []string{"minikube"}
```
Try
```
os.Args = []string{"minikube", "start"}
```